### ISSUE TITLE
Handle "An operation on function ... is already in progress" error

### ISSUE THREAD

**Issue by gr2m** [probot/example-google-cloud-function#8]
@bcoe do you think there is anything we could do to handle this error? It occurs if I merge multiple pull requests in quick succession. I don't think it's a big deal as it won't occur often and the message is clear enough, but maybe there is something simple we could do about it?

**Comment by bcoe** [comment]
@gr2m it's the deploy step that errors out?

**Comment by gr2m** [comment]
Yes: https://github.com/probot/example-google-cloud-function/actions?query=workflow%3ADeploy
